flink-runtime/src/main/java/org/apache/flink/runtime/executiongraph/failover/flip1/FixedDelayRestartBackoffTimeStrategy.java

```diff
@@ -57,6 +58,11 @@ public long getBackoffTime() {
 		return backoffTimeMS;
 	}
 
+	@VisibleForTesting
+	int getMaxNumberRestartAttempts() {
```

Comment:
I think it's reasonable to make this part of the public API, i.e., remove `@VisibleForTesting` and add `public`. Generally speaking, if a value is exposed by `toString()`, the value should be accessible by a getter or other methods.

Reply:
Ok.